adr/structure_classes.md

```diff
@@ -0,0 +1,28 @@
+# Structure of Classes Beyond Standard RailsMVC
+
+There are many ways to structure classes beyound the standard rails Model View Controller (MVC) structure, therefore to help make the project maintainable in the future we have come up with the following standards.
+
+* Avoid Helpers
+  * The rational behind this decision is standard rails helpers are hard to test efficiently and are global so they can have unintended consequences.
+  
+* Use a Decorators directory `app/decorators`
+  * Decorators should be used for read only operations that modify the data for viewing purposes.
+
+* Use a ChangeSet directory `app/change_set`
+  * ChangeSet should be used for reformatting the data for purposes of showing it to user/system for modification.
+  * We have kept the chnage set concept since these classes can also be useful for data import not just user facing formas.
+
+* Use a Services directory `app/services`
+  * Services should be used for encapsulating complex actions or actions that work across multiple models, like seeding or importing bulk data. These may change the underlying data structures.
+
+* Use a Mailer directory `app/mailer`
+  * Notifications will be put in the mailer directory so that we can stick with the standard defined by ActionMailer
+
+## Possible Future needs
+We currently believe that the buisness logic operations that are required for the system will be small enough to live directly on their primary model.  If in the future the models become to bloated we reccomend creating a BusinessRules directory `app/business_rules`.
+
+Currrently we plan to put the rules here:
```

Comment:
I would remove this section since it's very specific to tickets and upcoming implementations at this point in time, whereas I think the ADR should be more timeless.